sbin/pfdetect

```diff
@@ -156,10 +156,11 @@ sub _run_detector {
         my $line = $_;
         $logger->info("alert received: '$line'");
         my $data = $parser->parse($line);
-        unless($data) {
+        unless($data || $data eq 0) {
```

Comment:
So he had a gun your head.
@julsemaan no more violence.

eq is for string compare.
== is for numeric compare.

That code is confusing those two things